sklearn/model_selection/tests/test_search.py

```diff
@@ -72,9 +72,7 @@
 from sklearn.model_selection.tests.common import OneTimeSplitter
 
 
-# Neither of the following two estimators inherit from BaseEstimator,
-# to test hyperparameter search on user-defined classifiers.
-class MockClassifier:
+class MockClassifier(ClassifierMixin, BaseEstimator):
```

Comment:
Right, but I think we explicitly wanted it not inherit from BaseEstimator, as some thirds party estimators will not use BaseEstimator nor depend on scikit-learn. Why do we need those? Can't we manually add tags etc if needed?

Reply:
You are absolutely right, implementing a `_get_tags` method that just returns the `estimator_type` should be enough for the tests to pass. I will implement this ASAP.

Thanks @rth for the suggestion!